stories/instantsearch.stories.js

```diff
@@ -27,4 +27,20 @@ storiesOf('InstantSearch', module)
         },
       },
     })
+  )
+  .add(
+    'with refresh to reload',
```

Comment:
This story used to be on its own but it makes more sense to have it in `instantsearch`.